### PR TITLE
DEMO-49 | Add getMusic mock endpoint

### PR DESCRIPTION
**Pull request by Tantawit**
### Change

- New module: musicModule
- New package: joi
- New mock endpoint: getMusic(id)

### CODE REVIEW

File: src/music/musics.controller.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        description: { type: 'string', example: 'Cool music by Ed She Run' },
+      },
+    },
+  })
+  @Get('/:id')
+  async getMusic(
+    @Param('id', new JoiValidationPipe(Joi.string().required()))
+    id: string,
+  ) {
+    return await this.musicsService.getMusic(id);
```
Comment (sseerrnn) [inline src/music/musics.controller.ts:33]: Should add response tag in this segment?

File: src/music/musics.service.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  constructor() {}
+
+  async getMusic(id: string) {
+    const dumpUrl =
+      'https://www.soundboard.com/handler/DownLoadTrack.ashx?cliptitle=Never+Gonna+Give+You+Up-+Original&filename=mz/Mzg1ODMxNTIzMzg1ODM3_JzthsfvUY24.MP3';
+    return {
+      name: 'Dump Music',
+      url: dumpUrl,
+      descrpition: 'Dump music for testing API',
+    };
```
Comment (karnjj) [inline src/music/musics.service.ts:14]: I think it should have artist and music cover image fields.

File: src/music/musics.controller.ts (hunk 1 above)
Comment (Tantawit) [reply src/music/musics.controller.ts]: Sure!
https://github.com/2110336-2565-2/sec2-group10-demo-backend/blob/79d1660500467570e65298cfbe4986f14962ef76/src/music/musics.controller.ts#L13-L35

**Review by sseerrnn — APPROVED** [review]

**Review by karnjj — APPROVED** [review]
LGTM

**Pull request merged**